site2/website/release-notes.md

```diff
@@ -1,6 +1,181 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.7.1 &mdash; 2020-11-25 <a id=“2.7.1”></a>
+
+#### Broker
+
+- Fix topic ownership is not checked when getting topic policy [#9781](https://github.com/apache/pulsar/pull/9781)
+- Fix Cannot consume using older subscriptions if max subscription limit is reached [#9758](https://github.com/apache/pulsar/pull/9758)
```

Comment:
I was wondering whether we need to refine the PR description, you can consult @Anonymitaet on it.